base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -2380,33 +2438,6 @@ fn cleanup_orphans<T: BlockchainBackend>(db: &mut T, orphan_storage_capacity: us
     db.delete_oldest_orphans(horizon_height, orphan_storage_capacity)
 }
 
-fn prune_database_if_needed<T: BlockchainBackend>(
```

Comment:
I prefer us keeping the logic here

Reply:
Reverting back to the previous version, logic is unchanged